src/context.rs

```diff
@@ -653,6 +653,77 @@ impl Context {
         }
     }
 
+    pub fn activate_credential(
```

Comment:
Hah! I had actually typed those, but then realized just now that I never committed that!

Reply:
Actually, are we missing some clippy lint here, I would've thought the CI should fail if something wasn't documented. Or was it because we have way too many types and most are not documented?

Reply:
https://github.com/parallaxsecond/rust-tss-esapi/blob/master/src/lib.rs#L21  😀

Reply:
oops 😅

Reply:
We have so much stuff undocumented so enable that would mean a serious amount of errors.